examples/tex/js/components/TeXBlock.js

```diff
@@ -32,7 +32,7 @@ class KatexOutput extends React.Component {
     this._timer = setTimeout(() => {
       katex.render(
         this.props.content,
-        this.refs.container,
+        this.container,
```

Comment:
You'll need to update the actual ref in render too to be a callback. Looks like you just missed this one.